JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R

```diff
@@ -22,6 +22,10 @@
 #' @param addLineAtOne Logical, should a black line be draw at BF = 1?
 #' @param bty List of three elements. Type specifies the box type, ldwX the width of the x-axis, lwdY the width of the y-axis.
 #' @param plotLineOrPoint String, should the main geom in the plot be a line or a point?
+#' @param pointShapesSmallN String, if \code{plotLineOrPoint == "point"} then this controls the shape aestethic.
```

Comment:
.. and for large N?

Reply:
for large N (> 60) people should use a line (e.g., `plotLineOrPoint = "line"`). I didn't invent that cut-off that's simply what was used before. At that point, the code automagically switches to using line types.

Reply:
Then `pointShape` is an okay name, right? Considering you never get points for > smallN you don't need to specify that